FIAT/expansions.py

```diff
@@ -109,6 +110,37 @@ def xi_tetrahedron(eta):
     return xi1, xi2, xi3
 
 
+class PointExpansionSet(object):
+    """Evaluates the point basis on a point reference element."""
+
+    def __init__(self, ref_el):
+        if ref_el.get_spatial_dimension() != 0:
+            raise Exception("Must have a point")
+        self.ref_el = ref_el
+        self.base_ref_el = reference_element.Point()
+
+    def get_num_members(self, n):
+        return 1
+
+    def tabulate(self, n, pts):
+        """Returns a numpy array A[i,j] = phi_i(pts[j]) = 1.0."""
+        if len(pts) > 0:
+            results = numpy.ones((1, len(pts)), type(pts[0]))
+            return results
+        else:
+            return []
```

Comment:
> I think this wants to just be (in all cases):
> 
> ```
> return numpy.ones((1, len(pts)))
> ```

Done in 0733dd8226794e85b727348fbe17768e4eabf6c6 

> Should you be asserting `n == 1`?

Actually since P0 is the only allowed element you have to assert `n == 0`. Added in a74b91cdd446b20523aa63c12f672c06441111b3